crates/sui-sdk/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 //!
 //! It aims at providing a similar SDK functionality like the one existing for
 //! [TypeScript](https://github.com/MystenLabs/sui/tree/main/sdk/typescript/).
-//! Sui Rust SDK builds on top of the [JSON RPC API](https://docs.sui.io/sui-jsonrpc)
+//! Sui Rust SDK builds on top of the [JSON RPC API](https://docs.sui.io/sui-api-ref)
```

Comment:
```suggestion
//! Sui Rust SDK builds on top of the [JSON RPC API](https://docs.sui.io/sui-api)
```
This should probably go here instead of dumping right into the reference.